src/main/java/com/internet/shop/model/User.java

```diff
@@ -1,12 +1,14 @@
 package com.internet.shop.model;
 
+import java.util.List;
 import java.util.Objects;
 
 public class User {
     private Long id;
     private String name;
     private String login;
     private String password;
+    private List<Role> roles;
```

Comment:
В UML рекомендовали использовать Set
```suggestion
    private Set<Role> roles;
```

Reply:
agree